Description: Workaround to avoid PM related issues with older kernel
Forwarded: not-needed
Author: Ricardo Salveti <rsalveti@linaro.org>

diff --git a/services4/system/omap4/sysutils_linux.c b/services4/system/omap4/sysutils_linux.c
index 3de5b16..b0d7ae7 100644
--- a/sgx/services4/system/omap4/sysutils_linux.c
+++ b/sgx/services4/system/omap4/sysutils_linux.c
@@ -202,7 +202,7 @@ IMG_VOID DisableSGXClocks(SYS_DATA *psSysData)
 
 #if defined(LDM_PLATFORM) && !defined(PVR_DRI_DRM_NOT_PCI)
 	{
-		int res = pm_runtime_put_sync(&gpsPVRLDMDev->dev);
+		int res = 0; // XXX TL 3.1 pm_runtime_put_sync(&gpsPVRLDMDev->dev);
 		if (res < 0)
 		{
 			PVR_DPF((PVR_DBG_ERROR, "DisableSGXClocks: pm_runtime_put_sync failed (%d)", -res));
